programs/keosd/main.cpp

```diff
@@ -93,6 +93,7 @@ int main(int argc, char** argv)
              opts.count("print-default-config")) {
             return 0;
          }
+         return EXIT_FAILURE;
```

Comment:
Seems like this should match `nodeos` `main.cpp`, which would be to return `-1` (`INITIALIZE_FAIL`).

Reply:
Sure, will do, although `EXIT_FAILURE` is a [platform standard]( https://en.cppreference.com/w/c/program/EXIT_status) while `INITIALIZE_FAIL` is a leap invention :-)

Reply:
@heifner do you think this should go to 3.2 first?

Reply:
I assume it is applicable to 3.1, so it goes there first then merged up to 3.2 and then to main.

Reply:
OK, should I duplicate the whole enum in keosd/main.cpp?

```
enum return_codes {
   OTHER_FAIL        = -2,
   INITIALIZE_FAIL   = -1,
   SUCCESS           = 0,
   BAD_ALLOC         = 1,
   DATABASE_DIRTY    = 2,
   FIXED_REVERSIBLE  = SUCCESS,
   EXTRACTED_GENESIS = SUCCESS,
   NODE_MANAGEMENT_SUCCESS = 5
};
```

It it was me I'd have it in a common header, but I don't know which one.

Reply:
Maybe `plugins/chain_interface/include/eosio/chain/plugin_interface.hpp` ?